### ISSUE TITLE
Masking out regions before stitching

### ISSUE THREAD

**Issue by fidel-schaposnik** [OpenStitching/stitching#94]
Is there a way to mask out certain regions within each image for the purposes of stitching (these could be defined e.g. as a list of rectangles)? In other words, can we filter the keypoints we use to match images in order to exclude those within certain boundaries?

My use case concerns stitching frames from a video with graphical overlays. Using this module as is results in keypoints concentrated mostly on the overlays, which of course are static and therefore result in no stitching at all...

**Comment by lukasalexanderweber** [comment]
Yes that is possible, but not yet a stable Feature of the stitcher class. Have a look at https://github.com/OpenStitching/stitching/pull/55. I want to get back on it after I merged the Testing branch
